content/projects/data-science-specific/data-visualisation/mobile-money-viz/_index.md

```diff
@@ -23,11 +23,11 @@ tags:
 title: Financial Services Use in Tanzania
 ---
 
-## Background
+## About DataSet
 
 The [training](training.csv) dataset contains demographic information and what financial services are used by approximately 10,000 individuals across Tanzania. This data was extracted from the FSDT Finscope 2017 survey and prepared specifically for this challenge.
 
-Each individual is classified into four mutually exclusive categories:
+### Each individual is classified into four mutually exclusive categories:
 
 - No_financial_services: Individuals who do not use mobile money, do not save, do not have credit, and do not have insurance
```

Comment:
These naming conventions seem pretty funny. Definately not PEP8.  I quickly checked the training.csv file and it looks like these words dont come up at all. So I'm confused... 


Are you asking the learner to classify them? Or is there something else going on here?

The heading also says that there are 4 categories and then it only lists 2.  This is confusing. Where are the other 2?

Reply:
Hi Sheena,
Thanks for your feedback. :).  I am finding it challenging to comprehend this particular question. Within the context of the dataset, there are four categories that have been outlined. 
No_financial_services: Individuals who do not use mobile money, do not save, do not have credit, and do not have insurance
Other_only: Individuals who do not use mobile money, but do use at least one of the other financial services (savings, credit, insurance)
Mm_only: Individuals who use mobile money only
Mm_plus: Individuals who use mobile money and also use at least one of the other financial services (savings, credit, insurance)

Are you talking about these categories?

I would greatly appreciate your guidance on this matter.

Reply:
My mistake. I didn't see the other 2.

Naming conventions are still a problem though. Aim to follow PEP 8

ThisIsAClassName
this_is_a_variable_name
this_also_works_for_functions
THIS_IS_A_CONFIG_LIKE_CONSTANT
This_is_to_be_Avoided